cmd/version/version_test.go

```diff
@@ -31,8 +31,8 @@ var (
 
 func TestVersionCommand(t *testing.T) {
 	rootCmd := &cobra.Command{
-		Long:              "FlyteCTL is CLI tool written in go to interact with Flyteadmin service",
-		Short:             "FlyteCTL CLI tool",
+		Long:              "Flytectl is a CLI tool written in Go to interact with FlyteAdmin service",
```

Comment:
cli or CLI?

Reply:
Let's start with replacing all instances with CLI? And once we come up with a standard, we can change it accordingly..